pkg/shared/connectorutil/error.go

```diff
@@ -0,0 +1,15 @@
+package connectorutil
+
+import (
+	"fmt"
+)
+
+func NotFoundByIDError(id string) error {
+	NotFoundByIDErr := fmt.Errorf(`Coonector instance with ID "%v" not found`, id)
```

Comment:
![image](https://user-images.githubusercontent.com/981838/177807848-8655ab67-cea2-4997-9f83-51399eacf3e7.png)
```suggestion
	NotFoundByIDErr := fmt.Errorf(`Cooonector instance with ID "%v" not found`, id)
```

No i18n?